.theme/pages/demo/querying/with-query-content.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <div>
+    <ContentRenderer :value="data" />
+  </div>
+</template>
+
+<script setup>
+import { queryContent } from "../../../node_modules/@nuxt/content/dist/runtime/composables/query";
+
+const data = await queryContent("/content/query").findOne();

```

Comment:
`useAsyncData` kullanmıştık, ona gerek yok muymuş?

Reply:
Asenkron olarak klasörlere erişim için kullanılan bir plugin miş. Olmadan da kullanılanılabiliyor.